include/deal.II/dofs/dof_handler.h

```diff
@@ -25,6 +25,7 @@
 #include <deal.II/base/iterator_range.h>
 #include <deal.II/base/std_cxx11/shared_ptr.h>
 #include <deal.II/dofs/block_info.h>
+#include <deal.II/dofs/dof_accessor.h>
```

Comment:
This one was pretty hard to figure out. I think the answer is 'yes' (and this is what inspired the proposed blurb on style) but the template errors I got without this were unusually hard to understand.